lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -508,6 +536,21 @@ def test_update_thread_wrong_commentable_id(self, mock_get_discussion_id_map, mo
             mock_request
         )
 
+    def test_create_comment(self, mock_request):
+        self._setup_mock_request(mock_request)
+        self.setup_signal_handler(comment_created)
+
+        response = self.client.post(
+            reverse(
+                "create_comment",
+                kwargs={"course_id": self.course_id.to_deprecated_string(), "thread_id": "dummy"}
```

Comment:
I know you probably got it from somewhere else in this file, but the correct way to serialize a `CourseKey` is to write `unicode(self.course_id)`.